libraries/chain/apply_context.cpp

```diff
@@ -131,7 +131,7 @@ void apply_context::exec_one()
                }
             }
          }
-      } FC_RETHROW_EXCEPTIONS( warn, "pending console output: ${console}", ("console", _pending_console_output) )
+      } FC_RETHROW_EXCEPTIONS( warn, "${account}::${action} @ ${receiver} pending console output: ${console}", ("console", _pending_console_output)("account", act->account)("action", act->name)("receiver", receiver) )
```

Comment:
`cleos` outputs this as `${receiver} <= ${account}::${action}`. I'm not sure it matters much but would be interested if anyone has a preference. @arhag

Reply:
I like that.

Reply:
I don't have a strong preference how those three pieces of information should be presented. I agree it is a great developer experience improvement to include it though in whichever from. 

I'm wondering if we should also include the `action_ordinal` to disambiguate which one it comes from if there were multiple instances of the same action and receiver. But then again, even without this change, the transaction trace should already have enough information to determine which exact action within the trace the error came from.

Reply:
Changed to `${receiver} <= ${account}::${action}`.

I feel like the meaning of `action_ordinal` number would be hard to figure out for the developers, so would be more confusing than helpful. But maybe it's just me.